lib/android/app/src/main/java/com/reactnativenavigation/anim/NavigationAnimator.java

```diff
@@ -30,10 +39,12 @@ public NavigationAnimator(Context context) {
 
     public void animatePush(final View view, @Nullable final NavigationAnimationListener animationListener) {
         view.setVisibility(View.INVISIBLE);
-        ObjectAnimator alpha = ObjectAnimator.ofFloat(view, View.ALPHA, 0, 1);
-        alpha.setInterpolator(DECELERATE_INTERPOLATOR);
-
-        AnimatorSet set = new AnimatorSet();
+        AnimatorSet set;
+        if (options != null && !options.push.isEmpty()) {
```

Comment:
Lets try to avoid null as much as possible. We can init `options` and `push`, effectively utilising the null object pattern which contributes to a more predictable code.

Reply:
Furthermore, this logic should be extracted to `NavigationOptions`. Something like `options.getPushAnimation(view)`